adapters/appnexus/appnexustest/exemplary/simple-banner.json

```diff
@@ -1,6 +1,12 @@
 {
   "mockBidRequest": {
     "id": "test-request-id",
+    "app": {
```

Comment:
Shouldn't the `source` and `version` values be inside `app.ext.prebid`? `app` itself doesn't support a `prebid` field

Reply:
Updated the request in [ed9d44d](https://github.com/prebid/prebid-server/pull/2726/commits/ed9d44d7c86eb9356c76a8f2a9eceb4969e825ec)